### PR TITLE
Support for Django 3.x

### PR DESCRIPTION
**Pull request by philippbosch**
This change is a breaking change because it removes support for Python 2.x so a major release would be necessary. 

The Django team suggests to drop support of Django versions prior to 2.2 in their [release notes](https://docs.djangoproject.com/en/3.0/releases/3.0/#third-party-library-support-for-older-version-of-django).

### CODE REVIEW

**Comment by endersonmenezes** [comment]
It's correct :) I liked!

**Comment by endersonmenezes** [comment]
image
Fail only Python 2.7 :)

**Comment by endersonmenezes** [comment]
Need update on README now.

**Comment by philippbosch** [comment]
@endersonmenezes voilà!

**Comment by pmac** [comment]
Should also update the [tox.ini](https://github.com/pmclanahan/django-celery-email/blob/master/tox.ini) to remove Python 2.7 and Django 1.11, and add Django 3.0.

**Comment by philippbosch** [comment]
@pmac oh, right. just updated this. do you want me to also add to the changelog? if so, do you agree that we need a 3.0 for these changes?

**Comment by pmac** [comment]
If you don't mind updating the changelog that would be helpful, yes. And I do agree that this seems worthy of a major version number bump.

**Comment by philippbosch** [comment]
@pmac did some tweaks and clean-up, tests are passing, updated the changelog. i would say everything is in place for a release. let me know if anything is missing.

File: setup.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
     scripts=[],
     zip_safe=False,
     install_requires=[
-        'django>=1.8',
+        'django>=2.1',
```
Comment (pmac) [inline setup.py]: This should be `>=2.2`

File: requirements.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Django>=1.8
+Django>=2.1
```
Comment (pmac) [inline requirements.txt]: This should be `>=2.2`

**Comment by pmac** [comment]
Just a couple of small tweaks and then I think we're ready. Thanks again!!

**Comment by philippbosch** [comment]
oh yes, thanks for catching these. fixed now.

**Comment by pmac** [comment]
🎉

**Comment by pmac** [comment]
Release is up on PyPI. Thanks all! 🎉 

https://pypi.org/project/django-celery-email/3.0.0/

**Pull request merged**